miner/worker.go

```diff
@@ -821,8 +823,33 @@ func (w *worker) commitTransactions(env *environment, txs *transactionsByPriceAn
 			log.Trace("Not enough gas for further transactions", "have", env.gasPool, "want", params.TxGas)
 			break
 		}
+		// If we don't have enough blob space for any further blob transactions,
+		// skip that list altogether
+		if !blobTxs.Empty() && env.blobs*params.BlobTxBlobGasPerBlob >= params.MaxBlobGasPerBlock {
```

Comment:
shouldn't you check if `(1+env.blobs)*params.BlobTxBlobGasPerBlob` exceeds the max, rather than checking if we've already exceeded it?

Reply:
I used equality so that should mean that we reached the max

Reply:
Right!